src/web/components/Onwards/Carousel/Carousel.tsx

```diff
@@ -269,63 +188,53 @@ export const Title = ({
 	</h2>
 );
 
-const interleave = <A,>(arr: A[], separator: A): A[] => {
-	const separated = arr.map((elem) => [elem, separator]).flat();
-	if (separated.length > 0) separated.pop(); // remove separator at end
-	return separated;
-};
-
-type CardProps = {
-	trail: TrailType;
-	isFirst?: boolean;
+type CarouselCardProps = {
+	isFirst: boolean;
+	pillar: Theme;
+	design: Design;
+	linkTo: string;
+	headlineText: string;
+	webPublicationDate: string;
+	kickerText?: string;
+	imageUrl?: string;
 };
 
-const Card: React.FC<CardProps> = ({ trail, isFirst }: CardProps) => {
-	const kickerText = trail.design === Design.Live ? 'Live' : trail.kickerText;
-
-	return (
-		<a
-			href={trail.url}
-			className={isFirst ? cardWrapperFirstStyle : cardWrapperStyle}
-			data-link-name="article"
-		>
-			<img
-				className={cardImageStyle}
-				src={trail.image}
-				alt=""
-				role="presentation"
-			/>
-			<div className={headlineWrapperStyle(trail.design, trail.pillar)}>
-				<h4 className={headlineStyle(trail.design, trail.pillar)}>
-					{kickerText && (
-						<Kicker
-							text={kickerText}
-							design={trail.design}
-							pillar={trail.pillar}
-							showPulsingDot={trail.isLiveBlog}
-							inCard={true}
-						/>
-					)}
-					{trail.headline}
-				</h4>
-				<CardAge
-					webPublicationDate={trail.webPublicationDate}
-					showClock={true}
-					pillar={trail.pillar}
-					design={trail.design}
-				/>
-			</div>
-		</a>
-	);
-};
+export const CarouselCard: React.FC<CarouselCardProps> = ({
+	pillar,
+	design,
+	linkTo,
+	imageUrl,
+	headlineText,
+	webPublicationDate,
+	kickerText,
+	isFirst,
+}: CarouselCardProps) => (
+	<LI stretch={true} percentage="100%" showDivider={!isFirst} padSides={true}>
+		<Card
+			linkTo={linkTo}
+			format={{
+				display: Display.Standard,
+				design,
+				theme: pillar,
+			}}
+			headlineText={headlineText}
+			webPublicationDate={webPublicationDate}
+			kickerText={kickerText || ''}
+			imageUrl={imageUrl || ''}
+			showClock={true}
+			alwaysVertical={true}
+			minWidthInPixels={258}
+		/>
+	</LI>
```

Comment:
I guess now that this component doesn't do much, you could just move this code to line 394 and not have the component at all?